poker-pals/src/components/pokerComponents/RaiseToggle.vue

```diff
@@ -1,140 +0,0 @@
-<template>
-    <div class="raiseToggle"> <!--the container holding the raise input, it contains two buttons, a slider, and a value display-->
```

Comment:
Is this file not needed anymore?

Reply:
nope! replaced it with a number input (: